Virtusize/Sources/VirtusizeRepository.swift

```diff
@@ -322,16 +325,78 @@ internal class VirtusizeRepository: NSObject {
 		var mutualOrder = order
 		mutualOrder.externalUserId = externalUserId
 
-		let retrieveStoreInfoResponse = await VirtusizeAPIService.retrieveStoreInfoAsync()
-		if let storeInfo = retrieveStoreInfoResponse.success {
-			mutualOrder.region = storeInfo.region ?? "JP"
+		if let store = store {
+			mutualOrder.region = store.region ?? "JP"
 		} else {
-			throw retrieveStoreInfoResponse.failure ?? .unknownError
+			let retrieveStoreInfoResponse = await VirtusizeAPIService.retrieveStoreInfoAsync()
+			if let storeInfo = retrieveStoreInfoResponse.success {
+				mutualOrder.region = storeInfo.region ?? "JP"
+				store = storeInfo
+			} else {
+				throw retrieveStoreInfoResponse.failure ?? .unknownError
```

Comment:
If failing to retrieving the store info, would it crash the app?

Reply:
It should not crash the app as the errors are properly handled by `Virtusize.sendOrder`.